lib/averages.js

```diff
@@ -0,0 +1,10 @@
+function total(values) {
```

Comment:
you could also use the `reduce` method to help you out here - or just start the total at zero to avoid needed to use splice